.gitignore

```diff
@@ -13,4 +13,5 @@
 *.a
 mkmf.log
 
-spec/vcr_cassettes/drivers.yml
\ No newline at end of file
+spec/vcr_cassettes/drivers.yml
```

Comment:
That was my thought as well, but I'm happy to be safer than sorry if you think theres a chance we may end up with production data in cassettes (I'm VCR ignorant as well so I don't know possible it would be for this to happen).